host-interaction/network/create-network-socket.yml

```diff
@@ -0,0 +1,17 @@
+rule:
+  meta:
+    name: create network socket
+    namespace: host-interaction/network
+    author: alyssa.rahman@fireeye.com
+    scope: file
+    examples:
+      - 3db3e55b16a7b1b1afb970d5e77c5d98
+  features:
+    - and:
+      - os: linux
+      - or:
+        - import: recv
+        - import: recvfrom
+        - import: setsockopt
+        - import: socket
+        - import: sendto
```

Comment:
i think these routines are referenced in more specific ways by the rules found here: https://github.com/fireeye/capa-rules/tree/master/communication/socket

Reply:
agreed, vote to remove this generic rule